packages/plugins/src/utils/evm/constants.ts

```diff
@@ -2,6 +2,7 @@ import { ethereumNetwork } from '@sonarwatch/portfolio-core';
 import BigNumber from 'bignumber.js';
 
 export const ethFactor = new BigNumber(10 ** ethereumNetwork.native.decimals);
+export const ethFactorBigInt = 10n ** 18n;
```

Comment:
Would this work ?
Feels more aligned to the line right above.

```suggestion
export const ethFactorBigInt = 10n ** ethereumNetwork.native.decimals;
```